### PR TITLE
Change amplifier board

### PR DESCRIPTION
**Pull request by whatwareweb**
_Project proposals are no longer being accepted as of January 15th, 2023 23:59 EST._

If this a bug fix, documentation update, or other, please describe the purpose of this pull request below:

Update amplifier board to correct one

### CODE REVIEW

File: whatwareweb.md [hunk 1]
```diff
@@ -49,15 +49,15 @@ I am going to build a simple stereo amplifier. This amplifier will be able to pl
 | DC jack |  https://www.amazon.com/dp/B07XSQP3CT/ |  $4.88  |
 | Input selector switch | https://www.amazon.com/dp/B07JM3M2B4/ |  $13.49  |
 | Speaker terminals | https://www.amazon.com/dp/B08YMZ88Z5/ |  $6.39  |
-| Amplifier board | https://www.amazon.com/dp/B07WC2TK58/ |  $7.09  |
+| Amplifier board | https://www.amazon.com/dp/B07WC2TK58/ |  $21.99  |
```
Comment (bellesea) [inline whatwareweb.md:52]: hey, did you change your link? also, what's the reason for switching the amplifier board?

**Review by bellesea — CHANGES_REQUESTED** [review]

File: whatwareweb.md (hunk 1 above)
Comment (whatwareweb) [reply whatwareweb.md:52]: Yes I did, the reason is that I ordered the original boards (it came in a pack of 2) and they were both broken. I have returned them and chosen a different one with hopefully better QC.

Comment (bellesea) [reply whatwareweb.md:52]: ahhh ok cool!

**Review by bellesea — APPROVED** [review]
@whatwareweb this looks good but can you please update the amplifier board link? right now it's the same  https://www.amazon.com/dp/B07WC2TK58/

**Comment by whatwareweb** [comment]
Sorry about that, I pasted the wrong link. Should be fixed now

**Pull request merged**